src/App.scss

```diff
@@ -3,105 +3,19 @@
 html,
 body,
 #root {
+  -webkit-font-smoothing: antialiased;
```

Comment:
Please add minimal reset styles in this file. https://alligator.io/css/minimal-css-reset/